### DISCUSSION TITLE
2016/07/20/gaps-on-the-map-pueblo-colorado-to-cheyenne-wyoming-beacons

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#19] [Announcements]
# 2016/07/20/gaps-on-the-map-pueblo-colorado-to-cheyenne-wyoming-beacons

http://localhost:4000/2016/07/20/gaps-on-the-map-pueblo-colorado-to-cheyenne-wyoming-beacons

**Comment by jengalas** [comment]
Posted by Joe Gschwendtner
November 17, 2016 at 2:49 pm
_Imported from WordPress_

I believe I have located several missing from your inventory, one north of Castle Rock and another near Larkspur. The Larkspur unit is in a “People of Douglas County” history book and listed as on the Hunt Ranch Butte. The entry reads: “The beacon on Hunt Butte was powered by a gasoline engine operating on a time clock. Sometimes the clock got out of order and while it was being repaired, the engine had to be started by hand. Alice, Margaret, and I took turns riding horseback the eight or nine miles roundtrip morning and evening to start or shut the engine off”.

Could I get a copy of the Jeff Johnson map collage. If i had it, I might be able to locate more of the missing beacon/locations in my area?

All the best!

**Reply by jengalas** [reply]
Posted by Jeff Johnson
November 22, 2016 at 8:06 am
_Imported from WordPress_

Hi Joe,

You made a request for a “collage.” Sorry, but there isn’t any. I compared two airway charts, from 1928 and 1930, to estimate latitudes and longitudes. Zhanna has links to both. The only significant difference between the two that I’ve found concerns the beacons near or atop Castle Rock. The three Castle Rock beacons, 8, 8A, and 8B are in a short line roughly north-south, with 8B in the middle. I’m guessing from the article Zhanna found that 8A, at the south, may have been on top of the rock itself.

I was delighted that Zhanna immediately found two previously unknown arrows in southern Wyoming. I’m still kicking myself that I didn’t spot them in satellite views myself. In both cases, they are in plowed fields, and the farmers seem to be going out of their way to preserve the remnants: plowing around them, when they could probably save time and effort by just ripping them out.

Best of luck!

**Comment by jengalas** [comment]
Posted by Zhanna
July 24, 2016 at 8:19 am
_Imported from WordPress_

I can’t explain the mystery surrounding 8, 8A, and 8B. But I noticed that the nearby landmark Castle Rock looked like a good spot for a beacon. I found the following article from April 19, 1929:

> BEACON LIGHT IS INSTALLED ON TOP OF CASTLE ROCK
>
> The million-candle power beacon light which the government is having installed on top of Castle Rock, will be connected with the Commonwealth Company’s lines this week and put into service. This light is being installed in connection with the government’s Cheyenne-Pueblo air mail route to guide the mail pilots during their night flights. The beacon, from its vantage point on top of this historic rock will cast a beam of light which may be seen for many miles in all directions round about. Besides being of inestimable value to night flyers on the government air mail routes, the beacon will draw quite a lot of attention to Castle Rock from those passing this way.
